app/components/auk/button.hbs

```diff
@@ -7,9 +7,26 @@
   >
     <Auk::Loader />
   </button>
+{{else if @route}}
```

Comment:
Documenteer je deze nieuwe functionaliteit ook in de component z'n `.js`-file? Je hoeft niet alle argumenten van `LinkTo` verbatim over te nemen, maar één zin om te verduidelijken dat het om een extensie van `LinkTo` gaat die al dezelfde argumenten aanvaardt, is wel nodig.

Reply:
Goeie opmerking natuurlijk. Done!